quotefault/templates/bootstrap/additional_quotes.html

```diff
@@ -21,6 +21,58 @@
+                        </div>
+                        <div class="modal-body">
+                            <p>Are you sure you want to report this quote?</p>
+                        </div>
+                        <div class="modal-footer">
+                            <!-- TODO: Probably should not be using form and buttons and rather be using input. -->
+                            <!-- <a class="btn btn-primary btn-sm" href="/report/{{quote.id}}" id="report::{{quote.id}}" type="button btn-danger">Yes</a> -->
+                            <form method="POST" action="/report/{{quote.id}}" id="report::{{quote.id}}" method="POST">
+                                <button type="submit" class="btn btn-danger">Yes</button>
+                            </form>
+                            <button type="button" class="btn btn-success" onclck="reportClose({{quote.id}})" id="report_close_{{quote.id}}">No
+                            </button>
+                        </div>
+                    </div>
+                </div>
+            </div>
+            {% if metadata['uid'] == quote.submitter or metadata['uid'] == quote.speaker or metadata['is_admin'] %}
+                <button type="button" class="btn btn-warning btn-sm float-right" onclick="hide({{quote.id}})" id="hide_button_{{quote.id}}">Hide</button>
+                <div class="modal" id="hide_{{quote.id}}">
+                    <div class="modal-dialog" role="document">
+                        <div class="modal-content">
+                            <div class="modal-header">
+                                <h5 class="modal-title">Hide Quote</h5>
+                                <button type="button" class="close" data-dismiss="modal" aria-label="Close"><span
+                                    aria-hidden="true">&times;</span>
+                                </button>
+                            </div>
+                            <div class="modal-body">
+                                <p>Are you sure you want to hide this quote?</p>
+                            </div>
+                            <div class="modal-footer">
+                                <!-- TODO: Probably should not be using form and buttons and rather be using input. -->
+                                <!-- <a class="btn btn-primary btn-sm" href="/report/{{quote.id}}" id="report::{{quote.id}}" type="button btn-danger">Yes</a> -->
+                                <form method="POST" action="/hide/{{quote.id}}" id="hide::{{quote.id}}" method="POST">
+                                    <button type="submit" class="btn btn-danger">Yes</button>
+                                </form>
+                                <button type="button" class="btn btn-success" onclick="hideClose({{quote.id}})" id="hide_close_{{quote.id}}">No</button>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            {% endif %}
```

Comment:
Could we possibly deduplicate this  and share it with the above quotes?

Reply:
This is a copy-paste from the above quotes. Issue was this code was in the storage.html and not additional_quotes.html

Reply:
Can you put it in a separate file and include it to both?